core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1231,7 +1240,7 @@ public void writeIndexingBuffer() throws EngineException {
                 // The version map is using > 25% of the indexing buffer, so we do a refresh so the version map also clears
                 logger.debug("use refresh to write indexing buffer (heap size=[{}]), to also clear version map (heap size=[{}])",
                         new ByteSizeValue(indexingBufferBytes), new ByteSizeValue(versionMapBytes));
-                refresh("write indexing buffer");
+                refresh("write indexing buffer", false);
```

Comment:
I wonder if we can now always use this "refresh" path here rather than the `IndexWriter.flush` - refresh is now much lighter? (not sure what other side effects are there).

Reply:
+1 I think that's good